tests/contrib/funders/conftest.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Pytest configuration.
+
+See https://pytest-invenio.readthedocs.io/ for documentation on which test
+fixtures are available.
+"""
+
+import pytest
+
+from invenio_vocabularies.contrib.funders.resources import FundersResource, \
+    FundersResourceConfig
+from invenio_vocabularies.contrib.funders.services import FundersService, \
+    FundersServiceConfig
+
+
+@pytest.fixture(scope="module")
+def extra_entry_points():
+    """Extra entry points to load the mock_module features."""
+    return {
+        "invenio_db.models": [
+            "funders = invenio_vocabularies.contrib.funders.models",
+        ],
+        "invenio_jsonschemas.schemas": [
+            "funders = \
+                invenio_vocabularies.contrib.funders.jsonschemas",
+        ],
+        "invenio_search.mappings": [
+            "funders = \
+                invenio_vocabularies.contrib.funders.mappings",
+        ]
+    }
+
+
+@pytest.fixture(scope="function")
+def funder_full_data():
+    """Full funder data."""
+    return {
+        "scheme": "doi",
+        "id": "fund",
```

Comment:
moderate: this should potentially fail. The scheme is set to `doi` but the id value is not a DOI. We are not imposing any type of validation on this, which makes me think if we should:
a) Use `IdentifierScheme`, i.e. multiple inheritance (this could cause problems with `id` and `identifier`
b) Use only at validation time

For now this question can be shelved, because it will be completely different if decide to go for `identifiers` (plural). We have to wait for the datamodel to be ready.

Reply:
This is fixed now that there is an inner `identifiers` field. You can resolve the comment.